src/fe-small-advertisements/src/actions/name.ts

```diff
@@ -0,0 +1,16 @@
+export interface User {
+  name: string;
+}
+
+export const CHANGE_NAME = 'CHANGE_NAME';
+
+export interface ChangeNameAction {
+  type: typeof CHANGE_NAME;
+  payload: User;
+}
+
+export const changeNameActionCreator = ({
```

Comment:
Avoid using `ActionCreator` as suffix. Its gonna be used as argument in a dispatch function like this: 
```
dispatch(changeNameActionCreator({ name: newName }));
```

We know it is an action creator by the fact that is consumed from the `dispatch`

Reply:
It makes sense, it look cleaner. With `redux-toolkit` one can make it look even cleaner.